test/test-server/cli_handler.go

```diff
@@ -20,9 +20,15 @@ func handleFeedbacks(t *testing.T) http.HandlerFunc {
 		if len(*req.Content) > 20 {
 			w.WriteHeader(http.StatusForbidden)
 			errorJson, err := json.Marshal(&struct {
```

Comment:
Now that it works, can you define this struct, so we don't have to use `any`?

Reply:
I basically just copied the json error message string I use here from https://github.com/confluentinc/cli/blob/main/test/test-server/utils.go#L20-L25
In the usages of these strings, it seems more readable than what I'm doing here (also no extra json.Marshall() call). Wdyt?
``` 
w.WriteHeader(http.StatusForbidden)
errorJson, err := json.Marshal(ErrorJson{Errors: []ErrorInfo{{Detail: "feedback exceeds the maximum length", Status: "403"}}})
require.NoError(t, err)
_, err = io.WriteString(w, string(errorJson))
```
VS
```
w.WriteHeader(http.StatusForbidden)
_, err := io.WriteString(w, serviceAccountInvalidErrMsg)
```

Reply:
> https://github.com/confluentinc/cli/blob/main/test/test-server/utils.go#L20-L25

This should be avoided, since JSON strings can get out of hand and are really difficult to debug. For a really bad example, look at test/test-server/mds_routes_and_roles.go